base/util.jl

```diff
@@ -638,7 +638,7 @@ will always be called.
 function securezero! end
 @noinline securezero!(a::AbstractArray{<:Number}) = fill!(a, 0)
 securezero!(s::String) = unsafe_securezero!(pointer(s), sizeof(s))
-@noinline unsafe_securezero!{T}(p::Ptr{T}, len::Integer=1) =
+@noinline unsafe_securezero!(p::Ptr{T}, len::Integer=1) where T =
```

Comment:
Oups sorry. But if this is a style which we want for Base, why not enforce it by changing the syntax to disallow the absence of curly brackets in short form functions?